qiskit/aqua/components/variational_forms/variational_form.py

```diff
@@ -44,7 +45,9 @@ def __init__(self) -> None:
         self._bounds = list()  # type: List[object]
         self._preferred_init_points = None
         self._support_parameterized_circuit = False
-        pass
+        warn_package('aqua.components.variational_forms',
+                     'qiskit.algorithms.variational_forms',
```

Comment:
`VariationalForm` is no longer necessary at all and, thus, has been removed from Terra, too: https://github.com/Qiskit/qiskit-terra/pull/6064

Reply:
Fixed it